util/customTransforrmationsStore-v1.js

```diff
@@ -26,7 +27,13 @@ async function getTransformationCodeV1(versionId) {
     const response = await fetchWithProxy(
       `${getTransformationURL}?versionId=${versionId}`
     );
-    if (response.status !== 200) {
+
+    if (response.status >= 500) {
+      throw new CustomError(
+        `Error occured while fetching transformation with version id ${versionId}`,
+        809
+      );
+    } else if (response.status !== 200) {
```

Comment:
[check line below]Should we use custom error to also track response.status, may be it will provide some more info.

Reply:
done